CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # CHANGELOG
 
+## 1.29.0 (2022-09-16)
+
+MLflow 1.29.0 includes several major features and improvements
+
+Features:
+
+- [Tracking] Add support for load_text, load_image, load_dict to the fluent API (#6475, @subramaniam02)
+- [Tracking] Add attributes `creation_time` and `last_update_time` to the Experiment class (#6756, @subramaniam02)
+- [Tracking] Add a default generated run name to started runs if not supplied by user (#6736, @BenWilson2)
```

Comment:
```suggestion
- [Tracking] Automatically generate friendly names for Runs if not supplied by the user (#6736, @BenWilson2)
```

@BenWilson2 Can we move this to the top of the features list? It's just that great!